lib/resources/r4/financial_transaction.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name: FinancialTransaction
+field_name_base_url: http://hl7.org/fhir/R4/basic.html
+fields:
+- name: financial-transaction-method
```

Comment:
I think these names should be updated to:

extension(financial-transaction-method)
extension(financial-transaction-type)

Reply:
Also can you update the Account names in accordance with this?

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/4adf16f12daf8a74d8b1caef1c28f22dd3d0b8fd#diff-5125c86cde87f8439cb9d1bb859194e9R15

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/4adf16f12daf8a74d8b1caef1c28f22dd3d0b8fd#diff-5a59138a5bd3eaa9ab30d592af99e473R39

Reply:
+1